src/blocks/scratch3_sensing.js

```diff
@@ -313,12 +313,11 @@ class Scratch3SensingBlocks {
             }
         }
 
-        // Variables
+        // Local variables (not lists).
```

Comment:
I think the word "Local" here is misleading and should probably just say "Target's variables" or something

Reply:
Fixed!